sae_lens/analysis/neuronpedia_runner.py

```diff
@@ -269,49 +294,52 @@ def run(self):
                         )
                         feature_output["correlated_neurons_indices"] = (
-                            feature.left_tables_data.correlated_neurons_indices
-                        )
-                        feature_output["correlated_neurons_l1"] = self.round_list(
-                            feature.left_tables_data.correlated_neurons_l1
+                            feature.feature_tables_data.correlated_neurons_indices
                         )
+                        # TODO: this value doesn't exist in the new output type, commenting out for now
+                        # there is a cossim value though - is that what's needed?
+                        # feature_output["correlated_neurons_l1"] = self.round_list(
+                        #     feature.feature_tables_data.correlated_neurons_l1
+                        # )
                         feature_output["correlated_neurons_pearson"] = self.round_list(
-                            feature.left_tables_data.correlated_neurons_pearson
+                            feature.feature_tables_data.correlated_neurons_pearson
                         )
                         # feature_output["correlated_features_indices"] = (
-                        #     feature.left_tables_data.correlated_features_indices
+                        #     feature.feature_tables_data.correlated_features_indices
                         # )
                         # feature_output["correlated_features_l1"] = self.round_list(
-                        #     feature.left_tables_data.correlated_features_l1
+                        #     feature.feature_tables_data.correlated_features_l1
                         # )
                         # feature_output["correlated_features_pearson"] = self.round_list(
-                        #     feature.left_tables_data.correlated_features_pearson
+                        #     feature.feature_tables_data.correlated_features_pearson
                         # )
 
                     feature_output["neg_str"] = self.to_str_tokens_safe(
-                        vocab_dict, feature.middle_plots_data.bottom10_token_ids
+                        vocab_dict, feature.logits_table_data.bottom_token_ids
                     )
                     feature_output["neg_values"] = bottom10_logits
                     feature_output["pos_str"] = self.to_str_tokens_safe(
-                        vocab_dict, feature.middle_plots_data.top10_token_ids
+                        vocab_dict, feature.logits_table_data.top_token_ids
                     )
                     feature_output["pos_values"] = top10_logits
 
-                    feature_output["frac_nonzero"] = (
-                        feature.middle_plots_data.frac_nonzero
-                    )
+                    # TODO: don't know what this should be in the new version
+                    # feature_output["frac_nonzero"] = (
+                    #     feature.middle_plots_data.frac_nonzero
```

Comment:
here as well, not sure what the new name for this is, if it exists in the new version